### PR TITLE
Log Files to SQL Inserts PY parser

### PR DESCRIPTION
**Pull request by DevinSmithWork**
The following script converts any .log files in its directory into SQL code, which, when run, inserts new entries into UCOPReports/[Message Files] and UCOPReports.[Messages]

## Workflow:
• The .py script will run on all *.log files in its directory
• The filename and date are added to the UCOPReports.[Message File]
• The email addresses contained in the file are added to UCOPReports.[Messages], along with a reference to the [Message File] in which they were found.
• The INSERT statements for [Messages] can be rather long, so they are batched in groups of 1,000.

### CODE REVIEW

File: message_logs_to_sql_insert.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+	#--------------------------
+	# Output files
+
+	# Message Files SQL
+	sql_file = open (fname + "-insert.sql", "a")
+	table_name = "UCOPReports.[Message Files]"
+	column_names = "([File Name],[Email Date])"
+	sql_file.write("INSERT INTO " + table_name + "\n")
```
Comment (myucekul) [inline message_logs_to_sql_insert.py:81]: The script processes all the log file. Great. I'm looking for where it checks if a log file has already been processed?

File: message_logs_to_sql_insert.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+import glob
+import sys
+from datetime import datetime
+
+# Set to true to output email address CSV
+output_csv = False;
+
+# Gets all the log files in the folder where the python script is run
+print("Log files to process: ", glob.glob("*.log"), "\n")
```
Comment (myucekul) [inline message_logs_to_sql_insert.py:18]: This will also take the delegate log files too, right?

**Review by myucekul — COMMENTED** [review]
Looks good. I don't see much work needed to get this in crob job. Thanks Devin.

File: message_logs_to_sql_insert.py (hunk 1 above)
Comment (DevinSmithWork) [reply message_logs_to_sql_insert.py:81]: It doesn't check [Message Files] for duplicates -- this should be added before deploying (just to be on the safe side)

**Review by myucekul — APPROVED** [review]

**Pull request merged**